### PR TITLE
[EN/ZH] Update final result for `GBC 2022 Autumn`

### PR DESCRIPTION
**Pull request by RushFTK**
That's finally here. i have a long vacation when the tournament comes to ned

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by EdwardTsui — APPROVED** [review]
approved

File: wiki/Tournaments/GBC/GBC_2022_Autumn/en.md [hunk 1]
```diff
@@ -44,22 +44,25 @@ The **GB Cup 2022 Autumn** (***GBC 2022 Autumn***) is a 3v3 captain draft-based
 | *17-21st place* | ¥60 CNY |
 | *22-24th place* | ¥30 CNY |
 
+![](img/badge_official.png "GBC 2022 Autumn winner profile badge") ![](img/badge_1st.png "GBC 2022 Autumn 1st KanonBot profile badge") ![](img/badge_2nd.png "GBC 2022 Autumn 2nd KanonBot profile badge") ![](img/badge_3rd.png "GBC 2022 Autumn 3rd KanonBot profile badge")
+
 ## Organisation
 
 The GB Cup 2022 Autumn is run by the Team GB and various community members.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Autumn/en.md:51]:
```suggestion
The GB Cup 2022 Autumn was run by the Team GB and various community members.
```

**Review by yobrevelc — APPROVED** [review]

File: wiki/Tournaments/GBC/GBC_2022_Autumn/en.md [hunk 2]
```diff
@@ -44,22 +44,25 @@ The **GB Cup 2022 Autumn** (***GBC 2022 Autumn***) is a 3v3 captain draft-based
 | *17-21st place* | ¥60 CNY |
 | *22-24th place* | ¥30 CNY |
 
+![](img/badge_official.png "GBC 2022 Autumn winner profile badge") ![](img/badge_1st.png "GBC 2022 Autumn 1st KanonBot profile badge") ![](img/badge_2nd.png "GBC 2022 Autumn 2nd KanonBot profile badge") ![](img/badge_3rd.png "GBC 2022 Autumn 3rd KanonBot profile badge")
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Autumn/en.md:47]: why is there a winner badge *and* a first-place badge? what is "KanonBot"?

Comment (RushFTK) [reply wiki/Tournaments/GBC/GBC_2022_Autumn/en.md:47]: winner badge is images used in osu! website.  

`KanonBot` is a third-party chat bot for osu!, it also have it own badge system, player could display those badge on their profile like osu! website.  
like this:   
259UN6 {}7C@%XS_E8E(` P

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**